packages/angular-cli/models/webpack-build-common.ts

```diff
@@ -119,7 +119,13 @@ export function getWebpackCommonConfig(
       new webpack.LoaderOptionsPlugin({
         test: /\.(css|scss|sass|less|styl)$/,
         options: {
-          postcss: [ autoprefixer() ]
+          postcss: [ autoprefixer() ],
+          // workaround for https://github.com/webpack/css-loader/issues/340
+          context: path.resolve(__dirname, './'),
```

Comment:
Alternative: project root.

The reason is the `__dirname` is the path to _this_ file, but it might actually not be in the same directory than the project (for example, when you `npm link` the CLI). I think the correct solution is to use the project root.

Note that tutorial use `__dirname` because their `webpack.config.js` are at the root of their project. So in their case it's the valid thing to do.

Also, `path.resolve(__dirname, '.')` does nothing; just put the path in there.